### DISCUSSION TITLE
Utilizing burst2safe with local ASF Search resources

### DISCUSSION THREAD

**Discussion by sowens23** [ASFHyP3/burst2safe#89] [Q&A] [answered]
Hi William,

Excellent work on this project!

I have a question that might be straightforward. I'm a member of the ASF Intern team, and I was wondering if there's a way to bypass the requirement for providing Earthdata credentials or using .netrc authentication. Specifically, if I directly manage the .tiff and .xml downloads, would it be possible to use burst2safe to repackage the files that are already locally available?

Is there a way to access the source code directly to handle this, or any method to achieve this without the standard authentication process?

Thank you for your time!

**Comment by forrestfwilliams** [comment, marked as answer]
Hey @sowens23, yep I've just added a way to do this! Check out the new [local2safe](https://github.com/forrestfwilliams/burst2safe/blob/develop/src/burst2safe/local2safe.py) tool. Let me know if this fits your needs.

**Reply by sowens23** [reply]
This is great. Thank you so much.

**Comment by yakunhan** [comment]
Could you describe one example of local2safe usage?
